<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>671</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>211</width>
          <height>108</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="optRotacion">
           <property name="text">
            <string>Rotaci&amp;ón</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="optTraslacion">
           <property name="text">
            <string>Traslaci&amp;ón</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="optEscalado">
           <property name="text">
            <string>Esca&amp;lado</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="optReflexion">
           <property name="text">
            <string>Ref&amp;lexión</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>411</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkInter">
       <property name="text">
        <string>Interpolar</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frmEscalado">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Escala</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtEscala">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmTraslacion">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Traslación en y:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtY">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtX">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TRaslación en x:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmReflexion">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="chkX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reflejar en x</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reflejar en y</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmRotacion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Angulo:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtAngulo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>103</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
